src/templates/duplicate-modal.html

```diff
@@ -20,7 +20,9 @@ <h5 class='modal-title' id='duplicateModalLabel'>{{ 'Duplicate entry'|trans }}</
 
       {% if Entity.entityType.value == 'experiments' or Entity.entityType.value == 'items' %}
         <div class='d-flex justify-content-between'>
-          <label for='duplicateLinkToOriginal' class='col-form-label'>{{ 'Link to original experiment'|trans }}</label>
+          <label for='duplicateLinkToOriginal' class='col-form-label'>
+            {{ 'Link to original ' ~ (Entity.entityType.value == 'experiments' ? 'experiment' : 'item')|trans }}
```

Comment:
maybe we can use `Link to original entry`. `entry` being a generic term that encompass experiments and items. Also, I don't think this construction would be correctly detected by `poedit` to build the list of strings to translate.

Reply:
true, i was just realizing now that the modal entitles as duplicate an entry, which is way easier as a generic..
thank you